rust-runtime/aws-smithy-client/src/retry.rs

```diff
@@ -532,4 +565,34 @@ mod test {
         assert!(no_retry.is_none());
         assert_eq!(policy.retry_quota(), 480);
     }
+
+    #[test]
+    fn calculate_exponential_backoff_n_is_one() {
+        let n = 1.0;
+
+        for (expected, iter) in [(n, 0u32), (2.0, 1), (4.0, 2)] {
```

Comment:
this was surprisingly hard to load into my brain—does `.enumerate()` work here?

Reply:
yeah it does, I'll switch it